var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -75,3 +88,11 @@ def configure_args(self):
             args.append('--build=powerpc64le-redhat-linux-gnu')
 
         return args
+
+    @run_after('install')
+    def build_libsqlitefunctions(self):
+        if '+functions' in self.spec:
+            gcc = which('gcc')
```

Comment:
Hmm... I don't like this. A lot of people will build this without gcc. You should use `spack_cc` instead.